plugins/Live/API.php

```diff
@@ -146,6 +146,14 @@ public function getLastVisitsForVisitor($visitorId, $idSite, $filter_limit = 10,
     }
 
     /**
+     * Proxy for @see getLastVisitsDetails
+     */
+    public function getLastVisitsDetailsWidget($idSite, $period = false, $date = false, $segment = false, $countVisitorsToFetch = false, $minTimestamp = false, $flat = false, $doNotFetchActions = false)
+    {
+        return $this->getLastVisitsDetails($idSite, $period, $date, $segment, $countVisitorsToFetch, $minTimestamp, $flat, $doNotFetchActions);
+    }
```

Comment:
Not sure but why is there a proxy needed? Can we not use existing API method? That should be possible... let me know if there's a problem or help needed in order to use old API